src/libraries/System.Private.CoreLib/src/System/Collections/Generic/EqualityComparer.cs

```diff
@@ -180,7 +180,7 @@ public override int GetHashCode(byte b)
     // Needs to be public to support binary serialization compatibility
     public sealed partial class EnumEqualityComparer<T> : EqualityComparer<T>, ISerializable where T : struct, Enum
     {
-        internal EnumEqualityComparer() { }
+        public EnumEqualityComparer() { }
```

Comment:
This is a public API change, right?
I think it's not a problem - but it will probably come with "process"...

Reply:
> This is a public API change, right?

No. This type isn't really "public". It isn't exposed by a ref/contract assembly. https://docs.microsoft.com/en-us/search/?scope=.NET&terms=EnumEqualityComparer

It is just public in the implementation assembly because of this comment

https://github.com/dotnet/runtime/blob/82ca681cbac89d813a3ce397e0c665e6c051ed67/src/libraries/System.Private.CoreLib/src/System/Collections/Generic/EqualityComparer.cs#L180-L181

Reply:
The other "Needs to be public to support binary serialization compatibility" types in this file just depend on the compiler to generate the default public ctor. It would be nice to be consistent.

Reply:
Oh, ok - this one is special since it implements ISerializable.